### PR TITLE
[DCS-156] Feature/hermes clustering

### PR DESCRIPTION
**Pull request by anistal**
## Description
Definition of an skeleton that allows  to run Hermes using Akka Clustering. 

### Testing
You should test these scenarios:
* **Scenario 1)** With the purpose of to simulate an Akka Cluster a new test was added: ClusterHermesMultiJvmSpec.scala. This is a special test because it should be run using sbt.
You should follow these steps to execute the test:
   * Run sbt from the shell:
     ```shell
     $ ~/hermes/sbt
     ```
   * Pass this special test with this command:
     ```shell
     [info] Loading project definition from /home/anistal/Projects/genesis/project
     [info] Set current project to hermes (in build file:/~/hermes/)
     > multi-jvm:test
     ```
* **Scenario 2)** Manual testing running Hermes with three nodes. Steps:
   * Package generation.
     ```shell
     $ mvn clean package
     ```
   * Run a seed node (by default in the port 2551).
     ```shell
     java -Dakka.remote.netty.tcp.port=2551 -jar ~/hermes/target/hermes-0.1.0-SNAPSHOT-allinone.jar   
     ```
   * Run another two nodes (that will be use the first node as the seed).
     ```shell
     java -Dakka.remote.netty.tcp.port=2552 -jar ~/hermes/target/hermes-0.1.0-SNAPSHOT-allinone.jar
     ```
     ```shell
     java -Dakka.remote.netty.tcp.port=2552 -jar ~/hermes/target/hermes-0.1.0-SNAPSHOT-allinone.jar
     ```
  * Check that the seed node is running and that other nodes are connecting to the cluster (you could see easily in logs).

**Important note:** due to the difficulty of test an akka cluster and that the "special" test is not included in the coverage, it could be less than before.

- [x] Unit, integration, acceptance tests as appropriate
- [ ] Coverage (>90%)

### Documentation
- [x] Changelog update
- [x] Documentation entry

### Scalastyle
Result of scalastyle execution: No warnings or errors.

### CODE REVIEW

File: src/main/scala/com/stratio/hermes/runners/HermesRunner.scala [hunk 1]
```diff
@@ -16,10 +16,19 @@
...
  */
-object HermesRunner extends App {
+object HermesRunner extends App with HermesLogging {
 
-  def helloWorld(projectName: String): String = s"Hello $projectName!"
+  import com.stratio.hermes.implicits.HermesImplicits._
+  HermesRunnerHelper.welcome
 }
+
+
```
Comment (albertostratio) [inline src/main/scala/com/stratio/hermes/runners/HermesRunner.scala]: Could you please get rid of these blank lines?

File: src/multi-jvm/scala/com/stratio/hermes/multinode/ClusterHermesMultiJvmSpec.scala [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  val GenericTimeout = 20 seconds
+
+  import ClusterHermesMultiJvmSpec._
+
+  override def initialParticipants: Int = roles.size
+
+  val firstAddress = node(first).address
+  val secondAddress = node(second).address
+  val thirdAddress = node(third).address
+
```
Comment (albertostratio) [inline src/multi-jvm/scala/com/stratio/hermes/multinode/ClusterHermesMultiJvmSpec.scala]: Could you please put together all the vals of the class?

File: src/main/scala/com/stratio/hermes/runners/HermesRunner.scala (hunk 1 above)
Comment (albertostratio) [inline src/main/scala/com/stratio/hermes/runners/HermesRunner.scala]: Could you please get rid of these blank lines?

**Review by albertostratio — CHANGES_REQUESTED** [review]
Some minor changes. Good job!!

**Review by adoblas — APPROVED** [review]
LGTM

**Review by albertostratio — APPROVED** [review]

**Pull request merged**